clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -340,6 +344,8 @@ private KafkaAdminClient(AdminClientConfig config, String clientId, Time time, M
                      Metrics metrics, KafkaClient client, TimeoutProcessorFactory timeoutProcessorFactory) {
         this.defaultTimeoutMs = config.getInt(AdminClientConfig.REQUEST_TIMEOUT_MS_CONFIG);
         this.clientId = clientId;
+        LogContext logContext = new LogContext("[Consumer clientId=" + clientId + "] ");
```

Comment:
This should say `AdminClient`, not `Consumer`.

Reply:
Updated the log context.